src/Microsoft.Health.Fhir.Liquid.Converter/Microsoft.Health.Fhir.Liquid.Converter.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="Antlr4.Runtime.Standard" Version="4.8.0" />
     <PackageReference Include="DotLiquid" Version="2.0.366" />
+    <PackageReference Include="Hl7.Fhir.R4" Version="2.0.3" />
```

Comment:
Can we move the reference to the test project?

Reply:
Very reasonable, I'll do that.